### PR TITLE
Changelog attrs

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add docusaurus header to generated changelogs and regenerate them

### CODE REVIEW

File: docs/changelogs/v22.3.7.28-lts.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* Backported in [#37511](https://github.com/ClickHouse/ClickHouse/issues/37511): Fix logical error in normalizeUTF8 functions. Closes [#37298](https://github.com/ClickHouse/ClickHouse/issues/37298). [#37443](https://github.com/ClickHouse/ClickHouse/pull/37443) ([Maksim Kita](https://github.com/kitaisreal)).
+* Backported in [#37843](https://github.com/ClickHouse/ClickHouse/issues/37843): Fix segmentation fault in `show create table` from mysql database when it is configured with named collections. Closes [#37683](https://github.com/ClickHouse/ClickHouse/issues/37683). [#37690](https://github.com/ClickHouse/ClickHouse/pull/37690) ([Kseniia Sumarokova](https://github.com/kssenii)).
+* Backported in [#37940](https://github.com/ClickHouse/ClickHouse/issues/37940): Fix setting cast_ipv4_ipv6_default_on_conversion_error for internal cast function. Closes [#35156](https://github.com/ClickHouse/ClickHouse/issues/35156). [#37761](https://github.com/ClickHouse/ClickHouse/pull/37761) ([Maksim Kita](https://github.com/kitaisreal)).
+
+#### Bug Fix (user-visible misbehaviour in official stable or prestable release)
```
Comment (filimonov) [inline docs/changelogs/v22.3.7.28-lts.md:16]: same section twice for some reason...

**Review by filimonov — APPROVED** [review]

File: docs/changelogs/v22.3.7.28-lts.md (hunk 1 above)
Comment (Felixoid) [reply docs/changelogs/v22.3.7.28-lts.md:16]: misbehavior vs misbehaviour

File: utils/changelog/changelog.py [hunk 2]
```diff
@@ -306,7 +310,11 @@ def generate_description(item: PullRequest, repo: Repository) -> Optional[Descri
 
 
 def write_changelog(fd: TextIO, descriptions: Dict[str, List[Description]]):
-    fd.write(f"### ClickHouse release {TO_REF} FIXME as compared to {FROM_REF}\n\n")
+    year = date.today().year
+    fd.write(
+        f"---\nsidebar_position: 1\nsidebar_label: {year}\n---\n\n# {year} Changelog\n"
+        f"### ClickHouse release {TO_REF} FIXME as compared to {FROM_REF}\n\n"
```
[the remark this answers is not in the document]
Comment (Felixoid) [reply utils/changelog/changelog.py:316]: Actually, `docs/changelogs` is a directory for raw changelogs. Especially if we would add `return Description(item.number, item.user, item.html_url, item.title, category)`

As you see, the files definitely need some love from leather bags.

**Comment by rfraposa** [comment]
The docs do not use any of these files. I believe it's this file that needs updated: https://github.com/ClickHouse/ClickHouse/blob/master/CHANGELOG.md

**Comment by rfraposa** [comment]
If we can't automate adding the proper front matter to CHANGELOG.md, then we will just have to keep doing it manually. If that is the case, then we will need some kind of notification so we know when the changelog is modified and needs updating.

**Comment by Felixoid** [comment]
I've added `not for changelog` category too, so now we have it in raw changelogs

I am planning to regenerate v21 as well

**Comment by Felixoid** [comment]
@rfraposa I guess, github notification about a new release should do the task. You can subscribe to new releases on the top of the page 
image

**Pull request merged**